### PR TITLE
Denosoar GUI showcase

### PR DESCRIPTION
**Pull request by moha99ed**
Denosoar is an open source memory tool that is used to track potential memory leaks for applications built with Deno. Denosoar analyzes and displays in real time the RSS(resident set size), Heap Total, Heap Usage and External Memory using easy to read charts.

### CODE REVIEW

File: www/data/showcase.json [hunk 1]
```diff
@@ -228,6 +228,12 @@
     "link": "https://fresh-notion-blog.deno.dev/",
     "github": "xijaja/fresh-notion-blog",
     "image": "fresh-notion-blog"
+  },
+   {
+    "title": "Denosar",
+    "link": "https://https://denosoar.deno.dev//",
```
Comment (hashrock) [inline www/data/showcase.json]: could you fix this url?

Comment (moha99ed) [reply www/data/showcase.json]: Ah! just fixed it thanks

**Comment by hashrock** [comment]
could you add screenshots of your website?

https://github.com/denoland/fresh#adding-your-project-to-the-showcase

**Comment by moha99ed** [comment]
Denosoar_Screenshot

**Review by kt3k — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**